dvc/repo/experiments/save.py

```diff
@@ -21,24 +46,41 @@ def save(
 
     Returns the saved experiment's SHAs.
     """
-    queue = repo.experiments.workspace_queue
     logger.debug("Saving workspace in %s", os.getcwd())
 
-    staged, _, _ = repo.scm.status(untracked_files="no")
-    if staged:
+    _, _, untracked = repo.scm.status()
+    if untracked:
```

Comment:
I think we need to do something like:
```suggestion
    if include_untracked:
        untracked = [file for file in untracked if file not in include_untracked]
    if untracked:
```

Otherwise, we would be potentially raising the warning even if all the untracked files are in `include_untracked`